rcon/models.py

```diff
@@ -612,77 +612,6 @@ def to_dict(self) -> PlayerCommentType:
         }
 
 
-class ServerCount(Base):
```

Comment:
I wasn't sure if the corresponding database tables should be removed as well. However, keeping in mind that the feature itself might be interesting, I didn't want to add pointless migrations (just to add them in a later update). The tables should be empty anyway.

Reply:
I agree we should hang onto them in case we use them later. Some (very old) installs will have some computed data in there anyway that could be used in the future.